packages/typespec-python/src/emitter.ts

```diff
@@ -530,33 +516,41 @@ function hasDefaultStatusCode(response: HttpOperationResponse): boolean {
     return response.statusCode === "*";
 }
 
-function emitResponse(
-    context: SdkContext,
-    response: HttpOperationResponse,
-    innerResponse: HttpOperationResponseContent,
-    operation: Operation,
-): Record<string, any> {
+function emitResponse(context: SdkContext, response: HttpOperationResponse, operation: Operation): Record<string, any> {
     let type = undefined;
     let resultProperty = undefined;
-    if (innerResponse.body?.type) {
-        let modelType = undefined;
-        if (innerResponse.body.type.kind === "Model") {
-            const lroMeta = getLroMetadata(context.program, operation);
-            if (!hasDefaultStatusCode(response) && lroMeta) {
-                modelType = lroMeta.logicalResult;
-                if (lroMeta.finalStep?.target.kind === "ModelProperty") {
-                    resultProperty = lroMeta.finalStep.target.name;
-                }
-            } else {
-                modelType = getEffectiveSchemaType(context, innerResponse.body.type);
+    let body: Type | undefined = undefined;
+    const headers: Record<string, any>[] = [];
+    const lroMeta = getLroMetadata(context.program, operation);
+    if (!hasDefaultStatusCode(response) && lroMeta) {
+        body = lroMeta.logicalResult;
+        if (lroMeta.finalStep?.target.kind === "ModelProperty") {
```

Comment:
@msyyc what is the point of this code? i can't remember anymore. thanks!

Reply:
Seems introduced [here](https://github.com/Azure/autorest.python/pull/1821/files#r1178545663). But I don't get the point either. @msyyc could you give some example about this logic?

Reply:
Here is detailed context: https://github.com/Azure/autorest.python/pull/1821/files#r1247454845